docs/layers/arc-layer-64.md

```diff
@@ -0,0 +1,9 @@
+# Arc Layer 64
+
+The Arc Layer takes in paired latitude and longitude coordinated points and
```

Comment:
We should just say that this is a 64 bit version of the standard ArcLayer. And refer the reader there, and maybe to the article about 64 bits.